src/components/Modals/Receive/Receive.tsx

```diff
@@ -15,7 +15,7 @@ type ReceivePropsType = {
   accountId?: AccountSpecifier
```

Comment:
remove now this is unused?

Reply:
please see https://github.com/shapeshift/web/pull/2632/commits/cd018da5f83c7d4f1950ee150c899d3e10ca0709